compiler/src/dmd/backend/dtype.d

```diff
@@ -40,12 +40,14 @@ private __gshared
     int type_num,type_max;   /* gather statistics on # of types      */
 }
 
+type* tsclib;
```

Comment:
I don't think it is desired to make these thread local

Reply:
I had a similar concern, but my thinking was that I _think_ that not gshared is an improvement, though a minor one.  In practice, none of this code is threaded or has any likelihood of being thread safe any time soon.